app/views/posts/_post.html.erb

```diff
@@ -0,0 +1,8 @@
+<li id="post-<%= post.id %>">
+  <span class="content"><%= post.content %></span>
+  <span class="timestamp">
+    Posted <%= time_ago_in_words(post.created_at) %> ago.
+  </span>
+  <%= link_to "delete", post, method: :delete,
```

Comment:
- [ ] We need to check here first if the current user is the same as post author before displaying the delete button.
Don't forget to move this logic in helper method. [helpers](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers)